doc/api/http.md

```diff
@@ -1030,6 +1030,9 @@ By default, the Server's timeout value is 2 minutes, and sockets are
 destroyed automatically if they time out. However, if a callback is assigned
 to the Server's `'timeout'` event, timeouts must be handled explicitly.
 
+In recent versions of the Node.js 12.x, the default timeout can
+be overridden through the command line flag `--http-server-default-timeout`.
```

Comment:
I suggest to change these comments to something like:

```
It is possible to override the default timeout by using the
`--http-server-default-timeout` flag.
```

or

```
To change the default timeout use the `--http-server-default-timeout` flag.
```

Reply:
The part about Node 12 needs to remain.